README.md

```diff
@@ -1,32 +1,202 @@
-# Accessor and Mutator Methods
+# Scope
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Introduce Accessor (Getter) and Mutator (Setter) Methods
+- Use IntelliJ to Generate Getter and Setter Methods
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Encapsulation is a programming technique for keeping implementation
+details hidden from other classes.  This makes it easier to adapt
+and evolve code by isolating other classes from the internal state of a given class.
 
-## Topic 1
+We use the `private` keyword to encapsulate instance variables.
+A private variable is only visible within the class it is declared, so other classes can't see it.
+In Java, we define public methods to access or modify the value of a private instance variable.
+We can evolve the implementation of a public method if the class design changes, while keeping
+the method signature used by other classes the same.
+
+## Accessor and Mutator Methods
+
+
+The example programs we've worked with so far have involved
+as single class, where the `main` method is within the same class as the objects it creates
+and methods it calls.   But soon we will start to write programs that consist of several
+classes. So how can we access or modify object state if instance variables are private
+and only visible in a single class?
+
+In Java, we define public **accessor** and **mutator** methods to get and set private instance variables.
+
+- An accessor method takes no parameters and returns the value of a private instance variable.
+  The method return type matches the instance variable's declared type.
+  Accessor methods are also called **getter** methods.
+    - An accessor method for a non-boolean instance variable `abc` would be named `getAbc`.
+    - An accessor method for a boolean instance variable `abc` would be named `isAbc`.
+- A mutator method takes one parameter whose type and name match the instance variable's declared type
+  and name. Mutator methods don't return a value and are also called **setter** methods.
+    - A mutator method for instance variable `abc` would be named `setAbc` and would take a parameter named `abc`.
```

Comment:
"A mutator method for instance variable `abc`..."

Add an "an" between "for" and "instance"

Reply:
fixed